diff --git a/examples/keyboard.py b/examples/keyboard.py
index cfd8190..7f00a45 100644
--- a/examples/keyboard.py
+++ b/examples/keyboard.py
@@ -75,14 +75,14 @@ import uinput
 
 # Define mapping of capacitive touch pin presses to keyboard button presses.
 KEY_MAPPING = {
-                0: uinput.KEY_UP,    # Each line here should define a dict entry
-                1: uinput.KEY_DOWN,  # that maps the capacitive touch input number
-                2: uinput.KEY_LEFT,  # to an appropriate key press.
-                3: uinput.KEY_RIGHT, #
-                4: uinput.KEY_B,     # For reference the list of possible uinput.KEY_*
-                5: uinput.KEY_A,     # values you can specify is defined in linux/input.h:
-                6: uinput.KEY_ENTER, # http://www.cs.fsu.edu/~baker/devices/lxr/http/source/linux/include/linux/input.h?v=2.6.11.8
-                7: uinput.KEY_SPACE, #
+                0: uinput.KEY_A,    # Each line here should define a dict entry
+                1: uinput.KEY_B,  # that maps the capacitive touch input number
+                2: uinput.KEY_C,  # to an appropriate key press.
+                3: uinput.KEY_D, #
+                4: uinput.KEY_E,     # For reference the list of possible uinput.KEY_*
+                5: uinput.KEY_F,     # values you can specify is defined in linux/input.h:
+                6: uinput.KEY_G, # http://www.cs.fsu.edu/~baker/devices/lxr/http/source/linux/include/linux/input.h?v=2.6.11.8
+                7: uinput.KEY_H, #
               }                      # Make sure a cap touch input is defined only
                                      # once or else the program will fail to run!
 
